services/clamav/src/index.js

```diff
@@ -1,28 +1,45 @@
 /* istanbul ignore file */
-const NodeClam = require("clamscan");
+const axios = require("axios").default;
+const FormData = require("form-data");
+
 const fs = require("fs");
+const path = require("path");
+
+const { CLAM_AV_HOST } = process.env;
+
+module.exports = async (fileInformation, { fileName, location, debug }) => {
+  const host = CLAM_AV_HOST || "https://dev-clamav.azurewebsites.net";
+
+  try {
+    if (typeof fileInformation?.tempFilePath !== "undefined") {
```

Comment:
Why is this a string of "undefined" rather than a null or a falsy check?

Reply:
In javascript, you have to explicitly declare a null value to check against it which the tempFilePath. We can do a falsy check for undefined, null or false values but javascript also allows you to specify which types you are checking for explicitly using the above syntax, plus it was already there as well.